.github/ISSUE_TEMPLATE/epic.yml

```
@@ -0,0 +1,104 @@
+name: Epic
+description: Create an epic
+title: "Epic: "
+labels: ["type: epic"]
+body:
+- type: markdown
+  attributes:
+    value: Before raising an epic, please search for existing epics[[1](https://github.com/gitpod-io/gitpod/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+epic%22)][[2](https://github.com/gitpod-io/gitpod/issues?q=is%3Aopen+is%3Aissue+%22Epic%3A+%22)] to avoid creating duplicates. Read more about [Epics](https://www.notion.so/gitpod/Development-Process-2-0-6681854173ab4d2f92880f9f3d85cab5#321619f5a4bd4391be83c66feb2cdb49) (internal) in **Development Process**.
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary
+    description: TLDR description of the epic. Give a succinct and plain overview of what the epic is about.
+    placeholder: Give a succinct and plain overview of what the epic is about.
+  validations:
+    required: true
+- type: textarea
+  id: context
+  attributes:
+    label: Context
+    description: What thinking led to this? Provide any necessary historical context required to understand this epic.
+    placeholder: Provide any necessary historical context required to understand this epic.
+  validations:
+    required: true
+- type: textarea
+  id: value
+  attributes:
+    label: Value
+    description: Why should we do it? How do we know this is a real problem and worth solving?
+    placeholder: Explicitly describe the value to Gitpod and/or our users. I.e. why answer should we undertake this epic?
+  validations:
+    required: true
+- type: textarea
+  id: acceptance-criteria
+  attributes:
+    label: Acceptance Criteria
+    description: What needs to be done before the work is considered complete? The checks which must be complete for this epic to be considered done.
+    placeholder: Defines clearly when the work is complete. Acts as a litmus test for "done" and avoids "done" being ambiguous. Useful for implicit assumptions, e.g. ensuring docs updates are not forgotten.
+  validations:
+    required: true
+- type: textarea
+  id: measurement
+  attributes:
+    label: Measurement
+    description: How will we know whether we've been successful / solved the problem? How will you measure the success of the epic? Ideally this metric is one of our key product metrics.
+    placeholder: Important as it's how we track the outcomes (not just output) of the work and prove a change was worth it. Or it should be removed or iterated.
+  validations:
+    required: true
```

Measurement and hypothesis are quite closely linked, should we merge the two for simplicity? 

```suggestion
- type: textarea
  id: hypothesis-measurement
  attributes:
    label: Hypothesis & Measurement
    description: What behaviour change are we expecting to see in our user base? And how will that behaviour change be measured? Ideally this metric is one of our key product metrics.
    placeholder: Important as it's how we track the outcomes (not just output) of the work and prove a change was worth it. Or it should be removed or iterated.
  validations:
    required: true
```